prov/gni/test/sep.c

```diff
@@ -230,6 +227,9 @@ void sep_setup(void)
 					 FI_RECV | FI_READ);
 			cr_assert(!ret, "fi_ep_bind");
 
+			ret = fi_enable(tx_ep[i][j]);
+			cr_assert(!ret, "fi_enable");
+
```

Comment:
I'm confused by this.  Why does enabling tx_ep[i][j] have to come after binding to rx_ep[i][j]?